main.go

```diff
@@ -1052,11 +1053,27 @@ func main() {
 
 	mainLog.Info("Stop signal received.")
 
+	//if using async session writes stop workers
+	if config.Global().UseAsyncSessionWrite {
+		DefaultOrgStore.Stop()
+		DefaultQuotaStore.Stop()
```

Comment:
we do  only `DefaultQuotaStore.RemoveSession` so why does it need pool?

I am not even sure if `DefaultQuotaStore` is needed any more, I wasn't able to find its usage, might worth to double check if it is populated in Redis by some 3d party